pages/docs/config/properties/community.mdx

```diff
@@ -96,4 +96,107 @@ The generator attempts to make a _reasonable_ guess at the values in the config
 
 Other sections may be commented out but require some action by you. The best practice when configuring an Property is to read the comments, keys, and values within the generated file and fill in the appropriate values for your Property.
 
+### Aggregation Method and Filters
+
+Your Property config file will also contain `aggregationMethod` and `filter` options. Options for these may vary depending on your source type. All available options for your source will be listed in your config file template. Some common examples are:
```

Comment:
Maybe a little more info about where to get a template:

> All available options for your source will be listed in your config file template created when you [`grouparoo generate`](/docs/cli/config#generate). Some common examples are:

Reply:
sounds good.